src/test/java/com/thealgorithms/searches/agnosticBinarySearchTest.java

```diff
@@ -0,0 +1,27 @@
+package com.thealgorithms.searches;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class agnosticBinarySearchTest{
+
+    @Test
+    public void Testascending(){
```

Comment:
```suggestion
    public void testAscending(){
```
Please use camelCase for function names